src/components/Welcome/WelcomeQuestions.tsx

```diff
@@ -0,0 +1,39 @@
+import { WelcomeType } from "./WelcomeType";
+import { Accordion } from '@codegouvfr/react-dsfr/Accordion';
+import { fr } from "@codegouvfr/react-dsfr";
+
+export function WelcomeQuestions(props: { welcome: WelcomeType }) {
+
+    let caracteristiqueEnquete;
+
+    if (props.welcome.Enq_CaractereObligatoire && props.welcome.Enq_QualiteStatistique) {
+        caracteristiqueEnquete = ", reconnue d’intérêt général et de qualité statistique, est obligatoire";
+    }
+    else if (props.welcome.Enq_CaractereObligatoire && !props.welcome.Enq_QualiteStatistique) {
+        caracteristiqueEnquete = ' reconnue d’intérêt général est obligatoire';
+    }
+    else if (!props.welcome.Enq_CaractereObligatoire && props.welcome.Enq_QualiteStatistique) {
+        caracteristiqueEnquete = ", de qualité statistique, est reconnue d'intérêt général";
+    }
+    else {
+        caracteristiqueEnquete = " est reconnue d'intérêt général"
+    }
+
+    return (
+        <div className={fr.cx('fr-accordions-group')}>
+            <Accordion label="Quel est le cadre légal de l'enquête?">
+                <p>
+                    Vu l'avis favorable du Conseil national de l'information statistique, cette enquête{caracteristiqueEnquete}, en application de <a title="Loi n° 51 - 711 du 7 juin 1951 sur l'obligation, la coordination et le secret en matière de statistiques. - ouvre une nouvelle fenêtre" href={props.welcome.Loi_statistique} target="_blank" rel="noopener">la loi n° 51 - 711 du 7 juin 1951</a> sur l'obligation, la coordination et le secret en matière de statistiques.
```

Comment:
`target="_blank" rel="noopener"` causes a compilation error. [According to recommendations](https://mathiasbynens.github.io/rel-noopener/#recommendations), we should modify to `rel="noopener noreferrer"` in order to prevent security vulnerabilties in older browsers. 
see: 

- https://mathiasbynens.github.io/rel-noopener/#recommendations
- https://stackoverflow.com/questions/57628890/why-people-use-rel-noopener-noreferrer-instead-of-just-rel-noreferrer